analysis/filter/StructureModifier/src/test/kotlin/de/maibornwolff/codecharta/filter/structuremodifier/StructureModifierTest.kt

```diff
@@ -38,6 +54,7 @@ class StructureModifierTest {
         assertThat(errorStream.toString()).contains("invalid_project.cc.json is not a valid project")
     }
 
+    // TODO: Why is this disabled? Should it not be removed?
```

Comment:
I would remove this disabled test, since the last time it was touched was in 2020